public/app/features/alerting/unified/utils/rules.ts

```diff
@@ -142,3 +144,34 @@ export function alertStateToReadable(state: PromAlertingRuleState | GrafanaAlert
   }
   return capitalize(state);
 }
+
+export type FlatRule = {
```

Comment:
type should be in `types/unified-alerting.ts`, perhaps make a version of `RuleWithLocation`, eg `PromRuleWithLocation`